src/main/scala/net/degoes/00_intro.scala

```diff
@@ -54,9 +54,9 @@ object tour {
       val qworkers = List.fill(count)(qworker)
 
       (for {
-        fiber <- ZIO.forkAll(qworkers)
+        fiber <- ZIO.forkAll(qworkers.map(_.flatMap(_ => ZIO.unit)))
```

Comment:
Instead of this change, try: `qworkers.toList`

Reply:
@jdegoes does not help. "Cannot construct a collection of type List[A] ...."